channels/yandex-alice/src/main/kotlin/com/justai/jaicf/channel/yandexalice/api/AliceBotResponse.kt

```diff
@@ -5,18 +5,22 @@ import com.justai.jaicf.channel.yandexalice.api.model.Button
 import com.justai.jaicf.channel.yandexalice.api.model.Card
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
 import kotlinx.serialization.json.JsonObject
 
 @Serializable
 data class AliceBotResponse(
     var response: Response?,
     @SerialName("start_account_linking")
     var startAccountLinking: JsonObject? = null,
-    val session: Session,
-    val version: String
+    val version: String,
+    @Deprecated(message = "May be omitted")
+    val session: Session? = null,
+    val userStateUpdate: MutableMap<String, JsonElement?> = mutableMapOf(),
```

Comment:
It would be great to have some logic somewhere that propagates the request's session_state to this map.
Also I'd change the type of this field to `JsonObject` if possible.

Reply:
We disagree on that. The skills is going to be stateless as original and new states are provided by the API. If the framework provides logic to copy original state to the new one skill will loop forever in the same state. The new state even if its a copy of the original should be set explicitly by the developer. This is the reasoning behind having it in a separate field in the API.

Reply:
User state is incremental, meaning that the skill returns only increment to the user_state in response as it's shared among all surfaces of the user and there might be a race in changing it. That's why in the response MutableMap if used.
By design a developer has to pass `null` value associated with a key to remove the key's entry from the user state.
Details are explained here:
https://yandex.ru/dev/dialogs/alice/doc/session-persistence-docpage/